server/routes/testutils/appSetup.ts

```diff
@@ -136,107 +104,23 @@ function appSetup({
-    new SupportedPrisonsService(visitSchedulerApiClientBuilder, prisonRegisterApiClientBuilder, null)
-
-  app.use('/', indexRoutes(standardRouter(new MockUserService(), new MockSupportedPrisonsService())))
-
-  const auditService = auditServiceOverride || new AuditService()
-
-  app.use(
-    '/change-establishment/',
-    establishmentRoutes(
-      standardRouter(new MockUserService(), new MockSupportedPrisonsService()),
-      supportedPrisonsService,
-      auditService,
-      new MockUserService(),
-    ),
-  )
-
-  const prisonerSearchService =
-    prisonerSearchServiceOverride || new PrisonerSearchService(prisonerSearchClientBuilder, null)
-  const visitSessionsService =
-    visitSessionsServiceOverride ||
-    new VisitSessionsService(
-      prisonerContactRegistryApiClientBuilder,
-      visitSchedulerApiClientBuilder,
-      whereaboutsApiClientBuilder,
-      null,
-    )
-  app.use(
-    '/search/',
-    searchRoutes(
-      standardRouter(new MockUserService(), new MockSupportedPrisonsService()),
-      prisonerSearchService,
-      visitSessionsService,
-      auditService,
-    ),
-  )
-
-  const prisonerProfileService =
-    prisonerProfileServiceOverride ||
-    new PrisonerProfileService(
-      prisonApiClientBuilder,
-      visitSchedulerApiClientBuilder,
-      prisonerContactRegistryApiClientBuilder,
-      prisonerSearchClientBuilder,
-      supportedPrisonsServiceOverride ||
-        new SupportedPrisonsService(visitSchedulerApiClientBuilder, prisonRegisterApiClientBuilder, null),
-      null,
-    )
-  app.use(
-    '/prisoner/',
-    prisonerRoutes(
-      standardRouter(new MockUserService(), new MockSupportedPrisonsService()),
-      prisonerProfileService,
-      prisonerSearchService,
-      visitSessionsService,
-      auditService,
-    ),
-  )
-
-  const prisonerVisitorsService =
-    prisonerVisitorsServiceOverride || new PrisonerVisitorsService(prisonerContactRegistryApiClientBuilder, null)
-  const notificationsService = notificationsServiceOverride || new NotificationsService(notificationsApiClientBuilder)
-  app.use(
-    '/book-a-visit/',
-    bookAVisitRoutes(
-      standardRouter(new MockUserService(), new MockSupportedPrisonsService()),
-      prisonerVisitorsService,
-      visitSessionsService,
-      prisonerProfileService,
-      notificationsService,
-      auditService,
-    ),
-  )
-  app.use(
-    '/visit/',
-    visitRoutes(
-      standardRouter(new MockUserService(), new MockSupportedPrisonsService()),
-      prisonerSearchService,
-      visitSessionsService,
-      notificationsService,
-      auditService,
-      prisonerVisitorsService,
-      prisonerProfileService,
-      supportedPrisonsService,
-    ),
-  )
-  app.use(
-    '/visits',
-    visitsRoutes(
-      standardRouter(new MockUserService(), new MockSupportedPrisonsService()),
-      prisonerSearchService,
-      visitSessionsService,
-      auditService,
-    ),
-  )
+  if (services.userService === undefined) {
```

Comment:
Might look at mocking these two services by default in a different way once changes to `standardRouter` incorporated